torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -13,19 +13,31 @@ namespace nn {
 /// Options for the `Embedding` module.
 struct TORCH_API EmbeddingOptions {
   EmbeddingOptions(int64_t count, int64_t dimension);
-  /// The number of embeddings (number of rows in the table).
+  // The number of embeddings (number of rows in the table).
   TORCH_ARG(int64_t, count);
-  /// The size of each embedding vector (number of columns in the table).
+  // The size of each embedding vector (number of columns in the table).
```

Comment:
ditto for matching Python side: https://github.com/pytorch/pytorch/blob/9181b9c73e0772bc6d245f5f7f4a47771bef02bb/torch/nn/modules/sparse.py#L18